bin/neat

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
 
-require "neat"
+require File.dirname(__FILE__) + "/../lib/neat.rb"
```

Comment:
This change is seemingly unrelated. You can avoid this, and it's common to do something along these lines: `bundle exec ruby -I lib bin/neat`